internal/observability/middleware_test.go

```diff
@@ -532,3 +543,38 @@ func TestMiddlewareFailureSnapshot(t *testing.T) {
 	}
 	assert.Equal(t, want, snap, "Unexpected snapshot of metrics.")
 }
+
+func TestFixDefaultTransportValue(t *testing.T) {
+	tests := []struct {
+		desc      string
+		transport string
+		expected  string
+	}{
+		{
+			desc:     "unknown",
+			expected: "unknown",
+		},
+		{
+			desc:      "tchannel",
+			transport: "tchannel",
+			expected:  "tchannel",
+		},
+		{
+			desc:      "http",
+			transport: "http",
+			expected:  "http",
+		},
+		{
+			desc:      "grpc",
+			transport: "grpc",
+			expected:  "grpc",
```

Comment:
`desc` and `expected` are always the same - why not name all the subtests using `tt.expected`?

Reply:
done and moved to graph_test.go where it belongs